### PR TITLE
New update issue attribute script

### PR DESCRIPTION
**Pull request by weirandrewr**

### CODE REVIEW

File: update_issue_attributes.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+"""
+Delete purchases.
```
Comment (mattemerzian) [inline update_issue_attributes.py]: This is still referencing POs here

File: update_issue_attributes.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                            "value": value,
+                            "etag": etag
+                        }
+            }
+        }
+        api.request(request_body)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Bulk print location barcode labels.")
```
Comment (mattemerzian) [inline update_issue_attributes.py]: Barcode description in argument parser

Comment (mattemerzian) [reply update_issue_attributes.py]: I think I made this mistake in the PO one too

File: update_issue_attributes.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        for row in rows:
+            if issue_id == int(row[0]):
+                value = row[1]
+            continue
+
+    
+        request_body = {
+            "query": queries.UPDATE_ISSUE_ATTRIBUTE,
+            "variables": {
+                        "input": {
```
Comment (mattemerzian) [inline update_issue_attributes.py]: There appears to be too much indenting here

File: update_issue_attributes.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        )
+        issues = get_issues(ion_api)
+        update_issue_attribute(issues,ion_api)
+
+    except KeyError as e:
+        print(f"KeyError occurred: {e}")
+    except Exception as e:
+        error = f"Error occurred while running script: {e}"
+        print(error)
+        logger.exception(error)
```
Comment (mattemerzian) [inline update_issue_attributes.py]: It's customary to leave a blank line at the bottom of the file

File: update_issue_attributes.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+def update_issue_attribute(issues,api):
+    file = open("issues.csv")
+    csvreader = csv.reader(file)
+
+    rows = []
+    for row in csvreader:
+        rows.append(row)
+    file.close()
+
```
Comment (mattemerzian) [inline update_issue_attributes.py:43]: One blank line will do here for spacing b/t lines in a function if it is desired

**Review by mattemerzian — APPROVED** [review]
Looks good!

File: update_issue_attributes.py (hunk 2 above)
Comment (weirandrewr) [reply update_issue_attributes.py]: @mattemerzian hmm, looks like we never use that variable anyway, in either script? Can that line be removed?
edit: nevermind

**Pull request merged**